templates/pages/brands.html

```diff
@@ -8,6 +8,7 @@
 
 <main class="page">
     <h1 class="page-heading">{{lang 'brand.label'}}</h1>
+    {{{region name="page_content"}}}
```

Comment:
How can 2 regions in the same template placed at different locations have the same name ? We need need to be specific like `page_content_header` or `page_content_sidebar`.